docs/contributing/get-involved.md

```diff
@@ -2,4 +2,60 @@
+
+# 1. Joining the SourceCred Discord
+
+The very first step for getting involved with our community is to join our community discussions. Discord is the platform we use to be social and talk not only about our projects and goals, but also to get to know one another and build a sense of connection in real time.
+
+#### Step 1: Create a Discord Account
+
+If you have never used Discord before, you’ll want to download the application (or access it via web browser) and create your own account with their service. You can find step by step instructions for doing so [here] (insert link when guide is ready) (see "Join a Community Call link below).
+
+#### Step 2: Join SourceCred’s Community Discord Server
+
+Discord is a service, and SourceCred has it’s own little section of it where you can join the conversation. So once you’ve got Discord downloaded and your account set up, you’ll need to find us and join our community. For a step by step guide on how to do so, click [here] (insert link when guide is ready) (see "Join a Community Call link below).
+
+# 2. Poke Through Our Resources
+
+Eventually there will be dedicated channel in the Discord called “#signpost” or “#[read.me](http://read.me)” which will list all of the best beginner resources. For now, we’ve compiled a small list below of different types of documentation made by our community which may be useful to you. Not all of these documents are polished, or even finished, but it’s what we have available for now. Check out these resources (which are limited currently, but will be more polished and comprehensive by summer 2020) and get an idea for what SourceCred is all about, how we work, and what our values are.
+
+#### Beginner Resources:
+
+1. The “[Concepts](https://sourcecred.io/docs/concepts/cred)” Section of the SourceCred Website - this series of guides lays a foundation of the key terms and concepts that SourceCred is built upon like “Cred” and “Initiatives”.
+2. The [Code of Conduct ](https://sourcecred.io/docs/contributing/CODE_OF_CONDUCT) - a rough outlining of some of our community norms and values.
+3. The [Join a Community Call](https://discourse.sourcecred.io/t/how-to-use-discord-to-attend-a-community-call-proposal/725) rough draft - a protodocument that is still in the works but gives some basic information on how to join our weekly Community Call and how to join our Discord.
+4. The [Onboarding Machine](https://discourse.sourcecred.io/t/the-onboarding-machine-a-process-for-joining-sourcecred/726) Discourse Topic - Not so much a guide, but an explanation for the community of our thoughts on how to bring newcomers into our community effectively.
+5. The [How to Get an Ambassador](https://discourse.sourcecred.io/t/how-to-get-an-ambassador/727) How-To - a quick explanation of how to get person to person support as you enter the community and start contributing.
```

Comment:
For these -- leave the Discourse links, then replace with links to docs pages when we make them.

Reply:
Got rid of the section instead.